driver/driver.go

```diff
@@ -137,8 +137,10 @@ type MappingSources map[string][]struct {
 type ObjTool interface {
 	// Open opens the named object file. If the object is a shared
 	// library, start/limit/offset are the addresses where it is mapped
-	// into memory in the address space being inspected.
-	Open(file string, start, limit, offset uint64) (ObjFile, error)
+	// into memory in the address space being inspected. If the object
+	// is a linux kernel, relocationSymbol is the name of the symbol
+	// corresponding to the start address.
+	Open(file string, start, limit, offset uint64, relocationSymbol string) (ObjFile, error)
```

Comment:
Maybe `kernelRelocationSymbol` or (to shorten) `kernelRelocSym` or `kernelRelocSymbol`?

Trying to squeeze `kernel` in.

Reply:
I thought about that too, but the other parameters also only make sense for certain image types -- see "If the object is a shared library" comment above, so I'm inclined to leave as is.

Lmk if you feel strongly.

Reply:
Sgtm.